cmd/juju/cloud/add.go

```diff
@@ -137,6 +148,14 @@ func (c *addCloudCommand) runInteractive(ctxt *cmd.Context) error {
 		return errors.Trace(err)
 	}
 
+	pollster.VerifyURLs = func(s string) (ok bool, msg string, err error) {
+		err = c.Ping(provider, s)
```

Comment:
Do we need to put this behind a feature flag? Otherwise we won't be able to interactively bootstrap any of the providers where this is not yet implemented.